rust-tdshim/src/mp.rs

```diff
@@ -189,7 +191,8 @@ fn td_accept_pages(address: u64, pages: u64, page_size: u64) {
 }
 
 fn parallel_accept_memory(apic_id: u64) {
-    let mail_box = MailBox::new(get_mem_slice_mut(SliceType::MailBox));
+    // TODO: safe because we are the only user?
```

Comment:
What is difference between consumer and user?

I feel a better way is to let parallel_accept_memory() calls MP API.

We may file an issue and do that later.

Reply:
It's actually the same thing, just a marker for myself:)